src/chrondb/core.clj

```diff
@@ -18,7 +18,8 @@
     :age      ((faker/person :en) :age)}
    {:username ((faker/person :en) :username)
     :age      ((faker/person :en) :age)}])
-(def chrondb-struct-value
+
+(def chrondb-struct-value-merged
```

Comment:
here because had a rebinding the clj-kondo will broke, but you could ignore it adding this

`#_{:clj-kondo/ignore[:redefined-var]}
`